src/main/java/jp/co/future/uroborosql/exception/PessimisticLockException.java

```diff
@@ -36,7 +36,7 @@ public PessimisticLockException(final Throwable cause) {
 	}
 
 	private static String createMessage(final SqlContext context) {
-		return String.format("An error occurred due to pessimistic locking.\nExecuted SQL [\n%s]\nparams:%s",
+		return String.format("An error occurred due to pessimistic locking.%nExecuted SQL [%n%s]\nparams:%s",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.